benchmark/utils/general.hpp

```diff
@@ -195,20 +193,19 @@ void print_general_information(const std::string& extra)
 {
     std::clog << gko::version_info::get() << std::endl
               << "Running on " << FLAGS_executor << "(" << FLAGS_device_id
-              << ")" << std::endl
+              << ")\n"
```

Comment:
I don't think replacing `std::endl` with `\n` is necessary. Using `std::endl` is generally safer, and we don't care about the performance here.